server/src/main/java/io/crate/replication/logical/ShardReplicationChangesTracker.java

```diff
@@ -361,8 +362,10 @@ private void renewLeasesThenReschedule(long toSeqNoReceived) {
                                     ThreadPool.Names.LOGICAL_REPLICATION,
                                     () -> renewLeasesThenReschedule(toSeqNoReceived)
                                 );
+                            } else if (isClosed) {
+                                LOGGER.info("Exception renewing retention lease. Stopping tracking (closed=true)");
```

Comment:
Do we want to log in this case? cause tracking should stop anyhow if closed, so an exception on retention lease renewal maybe of no interest?

Reply:
I'm fine with either - or maybe we put it on debug level?

Reply:
Changed it to debug level